proxies/live/apiproxy/policies/AssignMessage.OAuthPolicyErrorResponse.xml

```diff
@@ -1,23 +1,27 @@
 <AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.OAuthPolicyErrorResponse">
   <Set>
     <StatusCode>401</StatusCode>
-    <Payload contentType="application/json">
+    <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
       {
         "resourceType": "OperationOutcome",
+        "meta": {
```

Comment:
Needs meta.lastUpdated. Should be current date time. Speak to Jack as he has to add for rate limiting.